recipes/pydoe/meta.yaml

```diff
@@ -0,0 +1,38 @@
+{% set version = "0.3.8" %}
+
+package:
+  name: pydoe
+  version: {{ version }}
+
+source:
+    fn: pyDOE-afc5949f01c010a26764adba9624871d51797270.tar.gz
+    url: https://github.com/tisimst/pyDOE/archive/afc5949f01c010a26764adba9624871d51797270.tar.gz
+    sha1: 54f87d106c3f49e1b4cff17a31ef8a9e0142ec5b
+
+build:
+  script: python setup.py install
```

Comment:
Please add the arguments `--single-version-externally-managed --record=record.txt` when using `setuptools`.